db/constraints/operations/create.py

```diff
@@ -1,34 +1,85 @@
-"""TODO This needs to be consolidated with db.constraints.base"""
 import json
-
 from db.connection import execute_msar_func_with_engine
-from db.constraints.utils import get_constraint_type_from_char, ConstraintType
-
-
-def create_unique_constraint(table_name, schema, engine, columns, constraint_name=None):
-    return execute_msar_func_with_engine(
-        engine,
-        'add_constraints',
-        schema,
-        table_name,
-        json.dumps([{'name': constraint_name, 'type': 'u', 'columns': columns}])
-    ).fetchone()[0]
-
+from db.constraints.utils import (
+    get_constraint_match_char_from_type, get_constraint_char_from_action,
+    get_constraint_type_from_char, ConstraintType
+)
 
-def create_constraint(schema, engine, constraint_obj):
-    with engine.begin() as conn:
-        return constraint_obj.add_constraint(schema, engine, conn)
 
+class Constraint():
```

Comment:
What's the point of this class? The methods are both more properly static (though they're set up as instance methods here), and its child classes don't actually inherit anything. 

I think you should remove it.

Reply:
Well, I think it should exist for the reason of not repeating the calling of the exact same SQL function from the separate constraint modules.